docs/userguide/storagedriver/overlayfs-driver.md

```diff
@@ -29,7 +29,7 @@ Since version 1.12, Docker also provides `overlay2` storage driver which is much
 more efficient than `overlay` in terms of inode utilization. The `overlay2`
 driver is only compatible with Linux kernel 4.0 and later.
 
-For comparison between `overlay` vs `overlay2`, please also refer to [Select a
+For comparison between `overlay` vs. `overlay2`, please also refer to [Select a
```

Comment:
The `.` is definitely incorrect here anyway. You don't put a dot unless there are omitted letters at the end. And between .. versus is wrong too. Use `and`.